tests/main/snap-service-start-timeout/task.yaml

```diff
@@ -0,0 +1,12 @@
+summary: Check that snaps can use start-timeout
+
+execute: |
+    dir="$TESTSLIB/snaps/test-snapd-service-start-timeout"
+
+    # with the 30s sleep, start-timeout stops the snap from working
+    ! snap try "$dir"
+
+    # drop the 'sleep 30'
+    sed -i -e '/@@@/d' "$dir/forking.sh"
```

Comment:
When another tests happens to use the same snap and ends up running after this one, the result may be a bit surprising.